interpreter/cling/include/cling/Interpreter/IncrementalCUDADeviceCompiler.h

```diff
@@ -6,10 +6,10 @@
 // of Illinois Open Source License or the GNU Lesser General Public License. See
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
-
 #ifndef CLING_INCREMENTAL_CUDA_DEVICE_JIT_H
 #define CLING_INCREMENTAL_CUDA_DEVICE_JIT_H
 
+#ifdef LLVM_HAS_NVPTX_TARGET
```

Comment:
This likely needs at least an include of the LLVM config to have somebody `#define` that macro